docs/authentication/enterprise-connections/overview.mdx

```diff
@@ -24,6 +24,15 @@ To configure subdomains for a SAML connection:
 > [!NOTE]
 > To enable the **Allow subdomains** option, your SAML connection domain must be an [eTLD+1](https://developer.mozilla.org/en-US/docs/Glossary/eTLD).
 
+#### Native applications
+
+Clerk ensures that security critical nonces are passed only to allowlisted URLs when the SAML flow is completed in native browsers or webviews. For maximum security in your **production** instances, you need to allowlist your custom redirect URLs via the [Clerk Dashboard](https://dashboard.clerk.com/) or the [Clerk Backend API](/docs/references/backend/redirect-urls/create-redirect-url).
+
+To allowlist a redirect URL via the Clerk Dashboard:
+
+1. In the Clerk Dashboard, navigate to the [**SSO connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
+1. Scroll to the **Allowlist for mobile OAuth/SAML redirect** section and add your redirect URLs.
```

Comment:
@NicolasLopes7 Is "Allowlist for mobile OAuth**/SAML redirect**" the new title of that section?

```suggestion
1. Scroll down to the **Allowlist for mobile OAuth redirect** section.
1. Under **Redirect URLs**, paste your the URL and select **Add**.
```

Reply:
Hi, it was but we've just changed to `SSO` https://github.com/clerk/dashboard/pull/4384/commits/4c32227264242d6f915bae3a5aac98d55a930f5d

Reply:
Thanks! I updated the suggestion.